arbitrum-docs/bold/concepts/bold-technical-deep-dive.md

```diff
@@ -0,0 +1,234 @@
+---
+title: 'BoLD: a technical deep dive'
+sidebar_label: 'Technical deep dive'
+description: 'A technical deep dive into the BoLD protocol.'
+user_story: 'As a user or researcher of the Arbitrum product suite, I want to understand the technical choices behind the BoLD protocol.'
+content_type: concept
+author: leeederek
+target_audience: 'Developers, users and researchers interested in the Arbitrum BoLD protocol.'
+sme: leeederek
+sidebar_position: 1
+---
+
+import PublicPreviewBannerPartial from '../../how-arbitrum-works/bold/partials/_bold-public-preview-banner-partial.md';
+
+
+<PublicPreviewBannerPartial />
+
+## Overview
+
+Under the hood, BoLD can offer time-bounded, permissionless validation because a correct Arbitrum state assertion is not tied to a single validator or entity. Instead, claims are tied to deterministic Merkle proofs and hashes that will be proven on Ethereum. Any party can bond on the correct state and, through interactive fraud proofs, prove their claim is correct. This means that a single honest party bonding on the correct state assertion will always win disputes, guaranteed.
+
+To put it simply, Arbitrum’s current dispute protocol assumes that any assertion that gets challenged must be defended against by each unique challenger. It is similar to a 1-vs-1 tournament, where the honest party may participate in one or more concurrent tournaments at any time. BoLD, on the other hand, enables an all-vs-all battle royal between two categories: Good vs Evil, where there must and will always be a single winner in the end.
```

Comment:
This paragraph is hard to read because the first two sentences describes the current protocol while the last sentence describes BoLD. I wonder if there's a way we could make the distinction more obvious. Nit though: I'm sure that anyone reading this section is paying close enough attention to understand what we're saying here.

Reply:
How about:

Arbitrum's current dispute protocol involves defending against challengers individually in a 1-vs-1 tournament setting. In contrast, BoLD enables an all-vs-all battle royale between Good and Evil categories, with a single winner always determined.
This dynamic is made possible by BoLD's time-bounded, permissionless validation using deterministic Merkle proofs and hashes. This allows any party to bond in the correct state and prove their claim through interactive fraud proofs. This ensures that a single honest party bonding on the correct state will always prevail in disputes.